docs/sources/auth/planning/index.md

```diff
@@ -122,6 +122,10 @@ Service account access tokens inherit permissions from the service account.
 
 ### 🚧 API keys
 
+Grafana API keys can be used to interact with data sources via HTTP APIs. API keys can have a well-defined and limited scope to resources with the help of [Roles].
```

Comment:
Should we point users to the service account docs as the way to do this? Eric has made some nice updates to those docs recently (not sure if merged in yet).